src/main/app/src/app/admin/inrichtingscheck/inrichtingscheck.component.html

```diff
@@ -41,7 +41,7 @@
             </mat-card-header>
             <mat-card-content>
                 <table *ngIf="!loadingZaaktypes" fxFlex="100" mat-table [dataSource]="dataSource" matSort #sort="matSort" matSortActive="valide"
```

Comment:
#sort="matSort" en matSortActive="valide" zijn niet meer nodig.

Reply:
Opgelost in opvolgende commit